src/rust/catpowder/win/runtime.rs

```diff
@@ -82,12 +88,11 @@ impl NetworkRuntime for SharedCatpowderRuntime {
 
     /// Transmits a packet.
     fn transmit<P: PacketBuf>(&mut self, mut pkt: P) -> Result<(), Fail> {
-        let header_size: usize = pkt.header_size();
-        let body_size: usize = pkt.body_size();
-        trace!("transmit(): header_size={:?}, body_size={:?}", header_size, body_size);
-
-        if header_size + body_size >= u16::MAX as usize {
-            let cause = format!("packet is too large: {:?}", header_size + body_size);
+        let outgoing_pkt: DemiBuffer = pkt.take_body().unwrap();
```

Comment:
This function returns a `Result`. Why should we `unwrap()` and `expect()` instead of failing?